.github/workflows/test.yml

```
@@ -26,32 +26,20 @@ jobs:
         uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
 
       - name: Setup Odin
-        uses: laytan/setup-odin@v2
+        uses: laytan/setup-odin
```

Please pin to a SHA. If you click on the commit of a release you can find it (see https://github.com/laytan/setup-odin/commit/41f9612bfec760bbb68b05b5747f319afe7c48d8)

```suggestion
        uses: laytan/setup-odin@41f9612bfec760bbb68b05b5747f319afe7c48d8
```

---

Thanks! Wasn't sure whether the `setup-oden` action could do that, but it seems to work correctly 👍